README.md

```diff
@@ -84,6 +84,37 @@ algorithms, including:
 * Primitive Methods: Repetition (REPET, 2DFT), Harmonic/Percussive, Vocal melody extraction, Timbre clustering
 * Benchmark Methods: High Pass filter, Binary Masks, Ratio Masks, Weiner Filtering
 
+## Interaction
+
+nussl supports light interaction via [gradio](https://www.gradio.app/).
+To launch a web interface where you can upload audio and hear
+separations, simply do:
+
+```python
+import nussl
+
+# Make a dummy signal to instantiate object.
+dummy_signal = nussl.AudioSignal()
+
+# Create some separation object with desired config.
+# This will use the 2DFT algorithm, but every method
+# in nussl has interaction implemented.
```

Comment:
How are these implemented? I assume they're all the same, right? I think this could be clearer.

Also, can you include an image of what it should look like here?

Reply:
Did you one better, added a gif showing off the interface.